Source/WebCore/dom/TreeScope.h

```diff
@@ -131,9 +132,11 @@ class TreeScope {
     std::span<const RefPtr<CSSStyleSheet>> adoptedStyleSheets() const;
     ExceptionOr<void> setAdoptedStyleSheets(Vector<RefPtr<CSSStyleSheet>>&&);
 
+    void addSVGResource(const AtomString& id, RenderSVGResourceContainer&);
     void addSVGResource(const AtomString& id, LegacyRenderSVGResourceContainer&);
     void removeSVGResource(const AtomString& id);
-    LegacyRenderSVGResourceContainer* svgResourceById(const AtomString& id) const;
+    RenderSVGResourceContainer* svgResourceById(const AtomString& id) const;
+    LegacyRenderSVGResourceContainer* legacysvgResourceById(const AtomString& id) const;
```

Comment:
How about:
lookupSVGResourceById and lookupLegacySVGResoureById?

Reply:
Sounds better, done.